### ISSUE TITLE
select('core/block-editor').getClientIdsOfDescendants is not working with single string

### ISSUE THREAD

**Issue by bigaru** [WordPress/gutenberg#54459]
### Description

According to [docs](https://developer.wordpress.org/block-editor/reference-guides/data/data-core-block-editor/#getclientidsofdescendants), `select('core/block-editor').getClientIdsOfDescendants` accepts `string` or  `string[]` as parameter. However, it only works with an array of string and not with a single string.  

When I looked into the [code](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-editor/src/store/selectors.js#L253), it seems that the argument `clientIds` is handled correctly.  Hence, I am even more confused.

### Step-by-step reproduction instructions

Note that the second time, the same argument is wrapped in an array:
```sh
# in dev console
wp.data.select('core/block-editor').getClientIdsOfDescendants('e6d611fd-1851-4500-9724-65f7906decf4')
> []
wp.data.select('core/block-editor').getClientIdsOfDescendants(['e6d611fd-1851-4500-9724-65f7906decf4'])
> ['db88ee65-0a80-4390-b054-a60521ed4748', '8cb4783b-cf78-4aa5-aacf-dea92c39b018', '574d41d3-595e-4f08-a791-1c15226643c5']
```

### Environment info

WP version 6.3.1
Google Chrome Version 115.0.5790.170

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by jordesign** [comment]
Hi @bigaru  - looping back to this issue - I wanted to see if this is still an issue you're seeing?

**Comment by Mamaduka** [comment]
The improvement was merged yesterday (#54421); the documentation for selectors is autogenerated based on the latest trunk. So this won't work until changes ship with WP 6.4.

**Comment by bigaru** [comment]
:open_mouth: ... now I feel stupid and everything makes sense now
(as an afterthought, maybe an option to select wp version on the documentation would be useful.)
